utils/fix_cmocka_xml.py

```diff
@@ -29,7 +29,8 @@
 parent_header = "<testsuites>\n"
 parent_footer = "</testsuites>\n"
 file_extensions = "*.xml"
-path = os.path.join(os.path.dirname(os.path.realpath(__file__)),"../test_results/")
+pwd = os.getcwd()
+path = os.path.join(pwd,"..","test_results") + os.path.sep
```

Comment:
I'm surprised pylint didn't complain about the lack of a space after the `,`s here.  The coding-style requires it.

No real reason not to just use `os.getcwd()` directly in line 33 rather than putting into a variable that's only used the once.

Reply:
Done.